ocs_ci/deployment/hub_spoke.py

```
@@ -7286,25 +7316,20 @@ def create_catalog_source(self, reapply=False, odf_version_tag=None):
 
         catalog_source_data = templating.load_yaml(constants.CATALOG_SOURCE_YAML)
 
-        if not config.ENV_DATA.get("clusters").get(self.name).get("hosted_odf_version"):
+        if not get_df_version(self.name):
             if not reapply:
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)